Sources/SkyFloatingLabelTextField.swift

```diff
@@ -539,7 +539,9 @@ public class SkyFloatingLabelTextField: UITextField {
     
     /// Invoked when the interface builder renders the control
     override public func prepareForInterfaceBuilder() {
-        super.prepareForInterfaceBuilder()
+        if #available(iOS 8.0, *) {
+            super.prepareForInterfaceBuilder()
+        } else {}
```

Comment:
+1 on what @RuiAAPeres said. Could you remove this please, and it will be good to go 👍